sdk/lib/opentelemetry/sdk/context.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    class Context < OpenTelemetry::Context
```

Comment:
This feels weird. Why do we need this subclass? The no-op propagator should be able to stand alone without it. 🤔 

OIC! You're just trying to mirror the structure from the API. I don't think we need to do that. We could really just make it a private module inside the `Configurator` class.

Reply:
Yeah, I was going for mirroring the structure rather than just dumping it somewhere. But if you're fine with putting it directly into the Configurator, that's fine with me - see e2a2219e .